packages/ui/Scrollable/Scrollable.js

```diff
@@ -348,6 +350,8 @@ class ScrollableBase extends Component {
 			vertical: true
 		};
 
+		this.overscrollEnabled = !!(props.applyOverscrollEffect);
```

Comment:
All member variables should be initialized before using it from the point of view of V8 engine optimization